main.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"os"
+
+	"github.com/ONSdigital/dp-dimension-importer/client"
+	"github.com/ONSdigital/dp-dimension-importer/config"
+	"github.com/ONSdigital/dp-dimension-importer/handler"
+	"github.com/ONSdigital/dp-dimension-importer/message"
+	"github.com/ONSdigital/dp-dimension-importer/repository"
+	"github.com/ONSdigital/go-ns/kafka"
+	"github.com/ONSdigital/go-ns/log"
+)
+
+var incomingKafka chan kafka.Message
+
+func main() {
+	log.Namespace = "dimension-importer"
+	cfg, err := config.Load()
+	if err != nil {
+		os.Exit(1)
+	}
+
+	log.Debug("Application configuration", log.Data{"config": cfg})
+
+	consumer, err := kafka.NewConsumerGroup(cfg.KafkaAddr, cfg.DimensionsExtractedTopic, log.Namespace, kafka.OffsetNewest)
+	if err != nil {
+		log.ErrorC("Could not create consumer", err, nil)
+		panic("Could not create consumer")
```

Comment:
Uses os.Exit(1) instead of panic. As you want to set the exit code to 1 (an error code in linux)